images/win/Windows2022-Readme.md

```diff
@@ -215,7 +213,7 @@ Note: MSYS2 is pre-installed on image but not added to PATH.
 ### Database tools
 - Azure CosmosDb Emulator 2.14.4.0
 - DacFx 15.0.5282.3
-- MySQL 8.0.27.0
+- MySQL 8.0.26.0
```

Comment:
Version downgrade is expected here (https://github.com/actions/virtual-environments-internal/issues/2999)